pkg/sentry/arch/arch_aarch64.go

```diff
@@ -32,26 +32,32 @@ import (
 const (
 	// SyscallWidth is the width of insturctions.
 	SyscallWidth = 4
+
+	// fpsimd_magic is the magic number which is used in fpsimd_context.
+	fpsimd_magic = 0x46508001
+
+	// fpsimd_context_size is the size of fpsimd_context.
+	fpsimd_context_size = 0x210
 )
 
 // aarch64FPState is aarch64 floating point state.
 type aarch64FPState []byte
 
-// initAarch64FPState (defined in asm files) sets up initial state.
-func initAarch64FPState(data *FloatingPointData) {
-	// TODO(gvisor.dev/issue/1238): floating-point is not supported.
-}
-
 func newAarch64FPStateSlice() []byte {
-	return alignedBytes(4096, 32)[:4096]
+	var size, align, capacity uint
+
+	size = uint(fpsimd_context_size)
```

Comment:
This doesn't need to go into a variable.

Reply:
Done.